sequelizeDatabase/sequelFunctions.js

```diff
@@ -400,23 +400,9 @@ modifyCompany = (CompanyID, alteredCompanyName) => {
 };
 
 //---------------Modify Fund---------------//
-modifyFund = (FundID, alteredFundName) => {
-  return new Promise((resolve, reject) => {
-    models.Funds.findOne({
-      where: {
-        FundID: FundID,
-      },
-    })
-      .then((fund) => {
-        fund
-          .update({
-            FundName: alteredFundName,
-          })
-          .then(resolve(true))
-          .catch(reject());
-      })
-      .catch(reject());
-  });
+modifyFund = async (FundID, alteredFundName) => {
```

Comment:
I'm a bit concerned about any potential breaks, but if it has not broken anything during testing, then I think we'll be good.